src/types.ts

```diff
@@ -45,3 +46,12 @@ export type OnchainKitProviderReact = {
   rpcUrl?: string;
   schemaId?: EASSchemaUid;
 };
+
+/**
+ * Note: exported as public Type
```

Comment:
I don't see this exported in the `index.ts`

Reply:
fixed in latest diff